Frontend/src/components/mentor/mentor-matches/MentorMatches.jsx

```diff
@@ -0,0 +1,82 @@
+import { useContext, useEffect, useState } from 'react';
+import { UserContext } from '../../../UserContext.jsx';
+import Request from '../mentor-requests/Request.jsx';
+import MatchCard from '../../mentee/mentee-matches/MatchCard.jsx';
+import './MentorMatches.css'
+import config  from '../../../../config.js';
+import MentorMatchCard from './MentorMatchCard.jsx';
+
+function MentorMatches() {
+    const { user } = useContext(UserContext);
+    const [mentorData, setMentorData] = useState(null);
+    const [requests, setRequests] = useState([]);
+    const [errorMessage, setErrorMessage] = useState('');
```

Comment:
There are several files have errorMessage and setErrorMessage, but I'm not seeing any error message displayed to user. Will you handle the errorMessage for all in future PR?

Reply:
I had these to just catch errors. I was told not to display error messages on the front end as it could reveal sensitive user information.